rollup.config.js

```diff
@@ -36,10 +39,11 @@ const config = {
       runtimeHelpers: true,
       ...babelConfig,
     }),
-    filesize(),
   ],
 }
-
+if (forBrowser) {
+  config.plugins.push(replaceNodeGlobals({}))
```

Comment:
We could also do it for all but cjs. As mjs seems to be what the ESM of Node.js would be. As we would also control the new type field of node 12.2.